api/nativeMessaging/main.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<!--
+ * Copyright 2013 The Chromium Authors. All rights reserved.  Use of this
```

Comment:
Remove these or update to 2021.

Reply:
I am not "Chromium Authors", can be removed.